File: Common/BaseApiCommon/InvokeParam.cs
using System;
using System.Runtime.Serialization;
namespace BaseApiCommon
{
    /// <summary>
    /// Class InvokeParam
    /// </summary>
    [DataContract]
    [Serializable]
    public sealed class InvokeParam
    {
        /// <summary>
        /// The interface
        /// </summary>
        [DataMember]
        public string Interface = string.Empty;
        /// <summary>
        /// The method name
        /// </summary>
        [DataMember]
        public string MethodName = string.Empty;
        /// <summary>
        /// The parameters
        /// </summary>
        [DataMember]
        public object[] Parameters = null;
    }
}

File: CS通信/HTTP/Client/JavaClient/javaClient/src/BaseApiCommon/InvokeParam.java
package BaseApiCommon;

import java.io.Serializable;

/**
 * Class InvokeParam
 */
// C# TO JAVA CONVERTER TODO TASK: Java annotations will not correspond to .NET
// attributes:
// [DataContract]
public class InvokeParam implements Serializable {
	public String Interface;
	public String MethodName;
	public Object[] Parameters;

}